### ISSUE TITLE
friendsandfoes-forge-mc1.19-1.5.5 crashes without a crashlog on forge 1.19 41.1.0

### ISSUE THREAD

**Issue by babacon88** [Faboslav/friends-and-foes#57]
### Minecraft version information

1.19

### Mod loader information

Forge

### Mod loader version information

41.1.0

### Mod version information

friendsandfoes-forge-mc1.19-1.5.5 

### Reproduction Steps

It crashes on startup with no crash log

### Crash Report and latest.log

there is no crash log, it was difficult for me to identify this mod caused crash
I also tried to load the mod itself alone

**Comment by Faboslav** [comment]
Hello, i just tested:
- mc 1.19 on forge 41.1.0 with friendsandfoes-forge-mc1.19-1.5.5 and architectury-5.12.42-forge
- mc 1.19.1 on forge 42.0.9 with friendsandfoes-forge-mc1.19-1.5.5 and architectury-5.12.42-forge
- mc 1.19.2 on forge 43.0.8 with friendsandfoes-forge-mc1.19-1.5.5 and architectury-5.12.42-forge

All of mentioned works just fine, if this is really friends&foes fault i will need more info to debug and fix the issue.

**Comment by Faboslav** [comment]
@babacon88 Is this still a problem?

**Comment by babacon88** [comment]
Hello there, the forge's new versions of 41.0.100 and above is incompatible with this mod
.63+ and .100 plus is known to have dratics change and incompatible many of older mods
will worth checking them out

**Comment by Faboslav** [comment]
I already fixed that with the last release. As mentioned above, `forge 41.1.0, 42.0.9 and 43.0.8`, which are the newest versions of the forge in the time of my testing works just fine, i need more info.

**Comment by GToidZ** [comment]
Launching Minecraft with this mod alone will ***obviously*** crash your game. This mod requires Architectury API (a dependency) to also be in the `mods` folder along with the mod.

**Comment by babacon88** [comment]
Yes my another forgetfull,.. But I did find the problem, still it didn't give me crash log when architectury API
The mod is not compatible with architectury-5.10.32-forge, released on 7/31, 2 weeks after friendsandfoes latest release, which was weird. I updated architectury api and now its working
I dont know if it is also incompatible with other versions older than architectury-5.12.42-forge (latest). If anyone having the same problem this worths a check out.
